app/controllers/response_controller.rb

```diff
@@ -148,6 +148,12 @@ def view
     set_content
   end
 
+  def view_review
```

Comment:
Added new view in response controller to be displayed when clicked on any review in the alternate view.